Makefile

```diff
@@ -314,8 +314,11 @@ $(BUILD_DIR)/src/overlays/%.o: CC := $(PYTHON) tools/asm_processor/build.py $(CC
 
 $(BUILD_DIR)/assets/%.o: CC := $(PYTHON) tools/asm_processor/build.py $(CC) -- $(AS) $(ASFLAGS) --
 else
+# The / is intentionally left out to capture other directories starting with "assets"
+$(BUILD_DIR)/assets%.o: CFLAGS += -fno-zero-initialized-in-bss -fno-toplevel-reorder
```

Comment:
still not a fan of this
not too sure what's the point either given the build rule
`$(BUILD_DIR)/assets/%.o: assets/%.c`
still is with-`/`

Reply:
The idea is to cover for people "less in the know", as it were, who may make a second asset dir for mods and then wonder why gcc is crashing on them. I think the flags override is less obvious than the makefile rules themselves but maybe that's not true, idk

Reply:
I get the idea but I do think its a bit messy to not be explicit. With the direction our assets are headed it should be sooner than later that custom asset dirs wouldnt be needed/encouraged

Reply:
I've added the `/` but left a comment which can hopefully be removed when assets are further along